conda-forge.yml

```diff
@@ -4,3 +4,5 @@ travis:
 appveyor:
   secure:
     BINSTAR_TOKEN: ipv/06DzgA7pzz2CIAtbPxZSsphDtF+JFyoWRnXkn3O8j7oRe3rzqj3LOoq2DZp4
+provider:
+  osx: circle
```

Comment:
@bekozi this should build OS X packages faster than Travis-CI but if we face any issues please let me know and we can revert this.

Reply:
Got it. Thanks!